### DISCUSSION TITLE
Partner has been waiting for a response for a suspended account for almost 4 weeks

### DISCUSSION THREAD

**Discussion by Bastian0930** [community/community#120860] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi. I just had a question - it looks like the average response time according to github is about a day for non urgent issues, but sometimes it can be a week or two. My partner is now at 4 weeks of waiting for an appeal - where could she go to ask for updates?

I've seen sometimes people ask about the ticketing system here, but if this is the inappropriate place, please tell me!

**Comment by ebndev** [comment]
Hi @Bastian0930,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
